src/java/org/apache/cassandra/service/ReadCallback.java

```diff
@@ -171,6 +207,11 @@ public void response(MessageIn<TMessage> message)
         }
     }
 
+    @VisibleForTesting
+    public void responseForTest() {
```

Comment:
Couldn't we just pass in dummy messages to `#response` to achieve the same result?

Reply:
yep!